Demo/Demo/Extensions/PaymentButtonEnums+Extension.swift

```diff
@@ -36,7 +36,7 @@ extension PayPalCreditButton.Color: CaseIterable {
 extension PaymentButtonEdges: CaseIterable {
 
     public static var allCases: [PaymentButtonEdges] {
-        [.hardEdges, .softEdges, .rounded, .custom(10)]
+        [.hardEdges, .rounded, .rounded, .custom(10)]
```

Comment:
.rounded is duplicated, is this intentional?

Reply:
hmm yea I'm not sure why `.rounded` shows up twice. Will update